editor/src/components/canvas/canvas-strategies/strategies/reparent-metastrategy.tsx

```diff
@@ -87,13 +87,25 @@ export function getApplicableReparentFactories(
       case 'REPARENT_AS_STATIC': {
         const fitness = result.isFallback ? 2 : 3
 
-        const targetParentDisplayType =
-          MetadataUtils.findElementByElementPath(
+        const childrenPaths = replaceContentAffectingPathsWithTheirChildrenRecursive(
```

Comment:
what if it has no children at this point? I think we should fall back to layoutSystemForChildren here too if it's available

Reply:
great idea, fixed in https://github.com/concrete-utopia/utopia/pull/3334/commits/9061d33830fd6e528efad4f611ca641d1a7ec414